README.md

```diff
@@ -1,5 +1,141 @@
-# curriculum-cr-training-contact-form
+<a name="readme-top"></a>
 
-This repo host an initial code for the tasks performed during the Code Reviewers training.
+<!-- TABLE OF CONTENTS -->
 
-Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 This repo host an initial code for the tasks performed during the Code Reviewers training.](#-this-repo-host-an-initial-code-for-the-tasks-performed-during-the-code-reviewers-training)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+  - [👥 Authors ](#-authors-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471). <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**Contact Form** is an application that allows users to send messages to the website owner. Built for the purpose of the code reviewer training program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">HTML</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **N/A**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> N/A
```

Comment:
- [ ] [OPTIONAL] Good work implementing some of the changes requested by the previous reviewer. Please fix your live demo link.  A broken live demo link makes the repo unprofessional. 🧑‍🏭